### ISSUE TITLE
Parameters of the problem

### ISSUE THREAD

**Issue by jiangllll** [Project-MONAI/research-contributions#28]
Hi, while I set pos_embed: str = "conv", should I need to change  conv_block: bool = False into  conv_block: bool = True?

**Comment by ahatamiz** [comment]
Hi @jiangllll 

Thanks for your comment. These indeed refer to different properties of the network. First, `pos_embed` denotes how the initial emebeddings are generated from the input images. One way to do it is to break the image into non-overlapping patches and use a perceptron layer to project into an embedding space. The other method is to just use a convolutional layer. Both techniques are equivalent but the implementation is different. We have provided both options by setting the `pos_embed` to `conv` or `perceptron`.

On the other hand, `conv_block` denotes whether or not to use convolutional blocks during the decoding process. By setting it true, these convolutional blocks are used in [UnetrUpBlock](https://github.com/Project-MONAI/MONAI/blob/d663d78e7579758047c5f83da9b0d0ef5ea0e4d5/monai/networks/blocks/unetr_block.py#L21) after the transposed convolutional layers. 

We have provided the settings that were originally used to train UNETR as the default parameters. 

Thanks
